src/pages/sunday-school-page.css

```diff
@@ -0,0 +1,6 @@
+#TeacherImage {
+    image-rendering: auto;
+}
+body {
```

Comment:
note that loading this CSS will probably conflict with the "main" style sheet [here](https://github.com/shaji618/mcnet/blob/main/src/index.css#L23-L28)

Reply:
Oh true. Let me kill this file for now then. It barely has anything in it at present and will make more sense to try and style things when I have a better idea of what the page will have.